test/markup/swift/strings.txt

```diff
@@ -0,0 +1,5 @@
+var string = """
+    var a = not actually code
+"""
+
+print(##"Use #" and "# to delimit a raw string."##)
```

Comment:
If we handle three variants (#, ##, ###) in the language then we'd optimally want all 3 listed here in markup tests also.